ui/admin/dashboard.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Sawarabi+Mincho" rel="stylesheet">
+    <link rel="stylesheet" href="../css/dashboard.css">
+    <title>Admin Dashboard</title>
+</head>
+
+<body>
+    <div class="admin">
+        <header class="admin-header">
+            <a href="../index.html" class="admin-logo">
+                <h2>Fast Food Dashboard</h2>
+            </a>
+            <div class="admin-bar">
+                <a href="#" class="logout">Log Out</a>
+            </div>
+        </header>
+        <nav class="sidebar">
+            <ul class="menu">
+                <li class="menu-item">
+                    <a href="../index.html" class="menu-link">Homepage</a>
+                </li>
+                <li class="menu-item">
+                    <a href="#" class="menu-link">Dashboard</a>
+                </li>
+                <li class="menu-item">
+                    <a href="#" class="menu-link">Create Food</a>
+                </li>
+                <li class="menu-item">
+                    <a href="orderlist.html" class="menu-link">List of All Orders</a>
+                </li>
+                <li class="menu-item">
+                    <a href="status.html" class="menu-link">Status of Orders</a>
+                </li>
+            </ul>
+        </nav>
+        <main class="statistics">
+            <h1>Statistics</h1>
+            <div class="stat">
+                <h2>Total Order Received</h2>
+                <p>Total: 500</p>
+            </div>
+            <div class="stat">
+                <h2 class="orange">Food Added</h2>
+                <p>Total: 700</p>
+            </div>
+            <div class="stat">
+                <h2 class="green">Total Order Accepted</h2>
+                <p>Total: 200</p>
+            </div>
+            <div class="stat">
+                <h2 class="red">Total Order Declained</h2>
+                <p>Total: 200</p>
+            </div>
+            <div class="stat">
+                <h2 class="orangep">Pending Orders</h2>
+                <p>Total: 100</p>
+            </div>
+        </main>
+    </div>
+
+</body>
+
+</html>
```

Comment:
please add a new line here
applies to all your prs and commits

Reply:
Working on it